syntax = "proto3";

package book;

service BookService {
  rpc GetBookDetails(GetBookDetailsRequest) returns (GetBookDetailsResponse);
}

message GetBookDetailsRequest {
  int64 item_id = 1;
}

message GetBookDetailsResponse {
  int64 user_id = 1;
  int32 progress = 2;
  double db_latency = 3;  // Latencia de la consulta a la base de datos
}
